inc/composer/class-docker-compose-generator.php

```diff
@@ -557,7 +558,7 @@ protected function get_service_tachyon() : array {
 				'environment' => [
 					'AWS_REGION' => 'us-east-1',
 					'AWS_S3_BUCKET' => "s3-{$this->project_name}",
-					'AWS_S3_ENDPOINT' => "https://{$this->tld}/",
+					'AWS_S3_ENDPOINT' => $this->set_url_scheme( "http://{$this->tld}/" ),
```

Comment:
Ah maybe this was the problem re. the S3 endpoint environment variable in the PHP container config. It might just need to match the new value.